### PR TITLE
Suite8beta

### PR DESCRIPTION
**Pull request by Mac-Rae**
Include changes for the Suite8 Wiki

### CODE REVIEW

File: content/8.x/admin/Compatibility Matrix.adoc [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+| Apache |2.2, 2.4 
+
+| IIS |8, 8.5
+
+2+^h| Database 
+
+| MariaDB |10.1+
+
+| MySQL |5.6 - 5.8
```
Comment (mattlorimer) [inline content/8.x/admin/Compatibility Matrix.adoc:38]: 5.6, 5.7, 8

**Review by mattlorimer — CHANGES_REQUESTED** [review]

File: content/8.x/admin/Compatibility Matrix.adoc (hunk 1 above)
Comment (Mac-Rae) [reply content/8.x/admin/Compatibility Matrix.adoc:38]: Will update now, Is there no longer an auto build on these docs?

Comment (mattlorimer) [reply content/8.x/admin/Compatibility Matrix.adoc:38]: takes time

Comment (Mac-Rae) [reply content/8.x/admin/Compatibility Matrix.adoc:38]: Ok sounds good

Comment (Mac-Rae) [reply content/8.x/admin/Compatibility Matrix.adoc]: > 5.6, 5.7, 8

Should be resolved now

**Review by mattlorimer — APPROVED** [review]

File: content/8.x/admin/Compatibility Matrix.adoc [hunk 2]
```diff
@@ -0,0 +1,49 @@
+---
+Title: Compatibility Matrix
+weight: 60
+---
+[NOTE]
```
Comment (pgorod) [inline content/8.x/admin/Compatibility Matrix.adoc:5]: For the notes this format can be used:
```
{{% notice note %}}
...
{{% /notice %}}
```
The same change can be made in the other files in this PR

**Pull request merged**